specification/iotcentral/resource-manager/Microsoft.IoTCentral/stable/2018-09-01/iotcentral.json

```diff
@@ -410,6 +410,43 @@
         "deprecated": false
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.IoTCentral/appTemplates": {
```

Comment:
We decided to make this global?

Reply:
we would firstly go with global, considering the customers can do separate CRUD operations on the templates

Reply:
We could also do crud if it's regional.